plugins/search/src/components/SearchBar/SearchBar.stories.tsx

```diff
@@ -16,21 +16,31 @@
 
 import { Grid, makeStyles, Paper } from '@material-ui/core';
 import React, { ComponentType } from 'react';
-import { SearchContextProviderForStorybook } from '@backstage/plugin-search-react';
+import {
+  searchApiRef,
+  MockSearchApi,
+  SearchContextProvider,
+} from '@backstage/plugin-search-react';
+import { ApiProvider } from '@backstage/core-app-api';
+import { TestApiRegistry } from '@backstage/test-utils';
 import { SearchBar } from './SearchBar';
 
 export default {
   title: 'Plugins/Search/SearchBar',
   component: SearchBar,
   decorators: [
     (Story: ComponentType<{}>) => (
-      <SearchContextProviderForStorybook>
-        <Grid container direction="row">
-          <Grid item xs={12}>
-            <Story />
+      <ApiProvider
+        apis={TestApiRegistry.from([searchApiRef, new MockSearchApi()])}
+      >
```

Comment:
Using the `TestApiProvider` here is a bit more convenient, lets you do just

```
<TestApiProvider apis={[[searchApiRef, new MockSearchApi()]]}>
```

Reply:
oh yes thats great, I'll update!